docs/guides/ORCHESTRATION_CHAT_COMPLETION.md

```diff
@@ -218,13 +218,11 @@ In this example, the input will be masked before the call to the LLM. Note that
 Change your LLM module configuration to add model parameters:
 
 ```java
-var llmConfig =
-    LLMModuleConfig.create()
-        .modelName("gpt-35-turbo")
-        .modelParams(
-            Map.of(
-                "max_tokens", 50,
-                "temperature", 0.1,
-                "frequency_penalty", 0,
-                "presence_penalty", 0));
+LLMModuleConfig llmConfig =
+    OrchestrationAiModel.GPT_4O.modelParams(
```

Comment:
this is a problem because we mutating the global static object. Changing it once will change it globally, which is rather unintuitive, since we are dealing with static constants here.

Reply:
agree.. `modelParams` should spawn a new instance to avoid global mutation.

Reply:
Done.